docker-for-windows/install.md

```diff
@@ -75,41 +72,35 @@ Looking for information on using Windows containers?
     It typically downloads to your `Downloads` folder, or you can run it from
     the recent downloads bar at the bottom of your web browser.
 
-2. Follow the install wizard to accept the license, authorize the installer, and
-   proceed with the install.
+2. Follow the instructions on the install wizard to accept the license, authorize the installer, and proceed with the install.
 
-    You are asked to authorize `Docker.app` with your system password during the
+    When prompted, authorize `Docker.app` with your system password during the
     install process. Privileged access is needed to install networking
     components, links to the Docker apps, and manage the Hyper-V VMs.
 
-3. Click **Finish** on the setup complete dialog to launch Docker.
+3. Click **Finish** on the setup complete dialog and launch the Docker Desktop app.
```

Comment:
Should `app` be `application`? Or maybe `and launch Docker Desktop`?

Reply:
Fixed in commit 165e8c21.